test/integration/ProjectBrowserTest.kt

```diff
@@ -84,51 +83,54 @@ class ProjectBrowserTest: IntegrationTest() {
             StudioState.project.current!!.reloadEntries()
             delayAndRecompose(composeRule)
 
-            composeRule.onNodeWithText(renamedFileName).assertExists()
+            nodeWithTextExists(composeRule, renamedFileName)
         }
     }
 
     @Test
     fun deleteAFile() {
-        studioTest(composeRule) {
-            cloneAndOpenProject(composeRule, source = SAMPLE_DATA_PATH, destination = testID)
+        runBlocking {
+            createData(source = SAMPLE_DATA_PATH, destination = testID)
+            openProject(composeRule, testID)
 
             StudioState.project.current!!.directory.entries.find { it.name == "file3" }!!.asFile().tryDelete()
             delayAndRecompose(composeRule)
 
             StudioState.project.current!!.reloadEntries()
             delayAndRecompose(composeRule)
 
-            composeRule.onNodeWithText("file3").assertDoesNotExist()
+            nodeWithTextDoesNotExist(composeRule, "file3")
```

Comment:
I'd go for `assertNodeNotExistsWithText`

Reply:
Implemented, propagated the `text =` naming as well.